marklogic-data-hub-central/ui/e2e/cypress/support/components/query/edit-queries-modal.tsx

```diff
@@ -0,0 +1,6 @@
+class EditQuery {
+
+
```

Comment:
@timur-isangulov , I don't see any integration or unit tests for edit or deleting a query, Any reason for that ?

Reply:
There is currently no functionality to add query on the unit test level or via UI because save query story is in development. I will add all the tests later when save query related stories are completed.

Reply:
Okay, makes sense to me, I see that you have created a seperate file for all the query api's, once your PR is merged I can add additional api's to that file. That would be helpful for mocking api's as well. So approving the PR.